### DISCUSSION TITLE
Debug build just causes menu to look very big

### DISCUSSION THREAD

**Discussion by Dante2202** [moudey/Shell#255] [Q&A] [answered]
I must be doing something wrong with the shell.nss file for the debug version because when I extracted the build and registered it with both the shell.nss file that came with the zip and when I replaced it with the working shell.nss file I was using I just get an over sized contex menu. 

Is there a default template I can use for the shell.nss that will work with the debug version?
![7BC64B8E-7420-4536-A25F-0153F2245C1C](https://github.com/moudey/Shell/assets/101029157/e3e0971b-1655-4429-a867-436546ad9b94)

**Comment by RubicBG** [comment, marked as answer]
Some of the settings are moved in imports\theme.nss. There 'font.size=20' is responsible for over sized context menu. if you remove this line it will use default settings like the release version.

One more tip: use 'name="modern"' instead of 'name="black"' 

I hope this helps
